src/util/debug.h

```diff
@@ -107,7 +107,13 @@ int rotate_debug_files(void);
 #define SSSDBG_TRACE_ALL      0x4000   /* level 9 */
 #define SSSDBG_BE_FO          0x8000   /* level 9 */
 #define SSSDBG_TRACE_LDB     0x10000   /* level 10 */
-#define SSSDBG_IMPORTANT_INFO SSSDBG_OP_FAILURE
+
+/* IMPORTANT_INFO will be logged if any of bits >=  OP_FAILURE are on: */
+#define SSSDBG_IMPORTANT_INFO (SSSDBG_OP_FAILURE|SSSDBG_MINOR_FAILURE|\
```

Comment:
`SSSDBG_OP_FAILURE` is 0b01000000. Maybe `(0xFFFFFF << 6)` instead?

Reply:
Or `~(SSSDBG_FATAL_FAILURE|SSSDBG_CRIT_FAILURE)`.
But why..?

Reply:
The whole purpose of named constants is to avoid using magic numbers.

Reply:
But `~(SSSDBG_FATAL_FAILURE|SSSDBG_CRIT_FAILURE)` isn't equivalent.

Reply:
`(~(SSSDBG_FATAL_FAILURE|SSSDBG_CRIT_FAILURE|0xF|0x800))` is closer, but is it any better..?

Reply:
No, the last one is even more complicated than mine:) Lets stick to your original change.

Reply:
Line 107 and 108 is out of scope of your change but has duplicated log level in comment.

Reply:
> Line 107 and 108 is out of scope of your change but has duplicated log level in comment.

This is intentional - see `debug_convert_old_level()`